spinn_utilities/conf_loader.py

```diff
@@ -48,15 +68,28 @@ def _logging_parser(self, config):
         return None
 
     def _machine_spec_parser(self, config):
-        """Load any cross-referenced machine specification file."""
+        """ Load any cross-referenced machine specification file.
+        """
         if not config.has_option("Machine", "machine_spec_file"):
             return None
         machine_spec_file_path = config.get("Machine", "machine_spec_file")
         config.read(machine_spec_file_path)
         return machine_spec_file_path
 
-    def load_config(self, config_parsers=[]):
+    def load_config(self, config_parsers=None):
+        """ Load the configuration
+
+        :param config_parsers:\
+            The parsers to parse the config with, as a list of\
+            (section name, parser); config will only be parsed if the\
+            section_name is found in the configuration files already loaded
+        :type config_parsers: list of (str, ConfigParser)
+        """
         config = ConfigParser.RawConfigParser()
+
+        # When the config is being loaded in a special environment, do nothing
+        if self._in_special_environment:
```

Comment:
Not sure if "in_special_environment" is right - maybe just "in_read_the_docs" simply to allow something else to happen when "in_travis" or "in_testing"?